src/types/homepage.ts

```diff
@@ -49,7 +49,7 @@ export interface InfopicSection {
   description?: string
   button?: string
   url?: string
-  images?: string
```

Comment:
Is this intended?

Reply:
Yup, the frontmatter for infopic uses `image` instead of `images`. The InfopicSection.jsx uses `image` too.